src/Repositories/MigrationRepository.php

```diff
@@ -43,6 +43,11 @@ public function delete(string $fileName): bool
             ->delete();
     }
 
+    public function clear(): void
```

Comment:
shall we maybe call it `truncate`? it's more explicit

Reply:
Done 👍